src/pages/workspaces/workspace/Notebooks.js

```diff
@@ -294,12 +304,8 @@ const Notebooks = _.flow(
       Ajax().Apps.app(app.googleProject, app.appName).delete()
     }
 
-    const createGalaxyInstance = () => {
-
-    }
-
     const applyGalaxyChanges = () => {
-      return app ? deleteGalaxyInstance() : createGalaxyInstance()
+      return app ? deleteGalaxyInstance() : this.setState({ creatingGalaxy: true })
```

Comment:
I wonder if we should make this state variable `modifyingGalaxy` or something? Since the panel will be used for more things?

Reply:
Another option is we actually make separate components with some shared styling for different viewModes, which might be cleaner if we are doing it from the get-go.

Reply:
Thinking more about it, I think we don't want separate components for the same reason we didn't do separate components for NewClusterModal.